packages/cli/lib/sandboxes.js

```diff
@@ -71,8 +77,10 @@ function getSandboxLimit(error) {
 
 // Fetches available sync types for a given sandbox portal
 async function getAvailableSyncTypes(parentAccountConfig, config) {
-  const parentPortalId = getAccountId(parentAccountConfig.portalId);
-  const portalId = getAccountId(config.portalId);
+  const parentId = getAccountIdentifier(parentAccountConfig);
+  const parentPortalId = getAccountId(parentId);
+  const id = getAccountIdentifier(config);
+  const portalId = getAccountId(id);
```

Comment:
Is `getAccountId()` redundant here if we already have the id?

Reply:
`getAccountId()` is not redundant for the deprecated config. It processes non-numerical account ids:

https://github.com/HubSpot/hubspot-local-dev-lib/blob/2d681abf5d087afd71a6e6a6f2bbbc15e21dbbe9/config/config_DEPRECATED.ts#L396-L436